stork_config/stork_config.go

```diff
@@ -0,0 +1,149 @@
+package main
```

Comment:
I'd suggest putting this into the stork CLI, this is an open source repository and it doesn't really belong here.

Reply:
Stork doesn't have a dedicated CLI and I really do not believe this belongs where the current CLI is.

Reply:
Just throw a linter into the stork repository as a binary, it's much more related to that repository than it is to `devbase` which is really just for common utilities between repositories

Reply:
Ok, I will put it there then.